src/BridgedRoom.ts

```diff
@@ -47,6 +47,9 @@ interface ISlackChatMessagePayload extends ISlackToMatrixResult {
     icon_url?: string;
 }
 
+const RECENT_MESSAGE_MAX = 10;
```

Comment:
10 seems on the low side? It's not like even 100 would use much ram?

Reply:
My worry is many active slack rooms would start to chug a bit, although 10 is a bit low. How about 25?

Reply:
What's going to slow it down? the `thing in list` operations? That doesn't seem like something that should be slow moving?

Reply:
I'm thinking of matrix.org and it's thousands upon thousands of rooms :(